neard/src/main.rs

```diff
@@ -146,6 +147,7 @@ fn main() {
         ("run", Some(args)) => {
             // Load configs from home.
             let mut near_config = load_config(home_dir);
+            validate_genesis(&near_config.genesis);
```

Comment:
This is good, but similar to above question - do we update genesis on the dump, removing extra locked / etc?
We do need to add more complex stuff into `state_migration` to catch such situation - for example send a staking transaction which would make `locked > 0` but wouldn't add to validators list.

Reply:
yes will address that in a separate PR.